history-network.md

```diff
@@ -286,6 +266,6 @@ To find a piece of content for `content-id`, a node performs a content lookup vi
 
 ### Storing Content
 
-To store a piece of content with key `content-id`, a node performs a lookup to find the `k` closest nodes with radii that contain `content-id`. Then the node sends `Offer` RPCs to those nodes. For any node that responds to the `Offer` RPC with a `Accept` RPC, the local node responds with a `Store` RPC for the content.
+To store a piece of content with key `content-id`, a node performs a lookup to find the `k` closest nodes with radii that contain `content-id`. Then the node sends `Offer` RPCs to those nodes. For any node that responds to the `Offer` RPC with a `Accept` RPC, the local node attempts to transmit the content over the uTP connection with the `connection-id` from the `Accept` RPC.
```

Comment:
Nitpick: I think that nowhere else in the specs the term RPC is used. It is typically `message`, `protocol messages` or `request/response message`. For consistency and thus clarity, lets avoid the term RPC. 

```suggestion
To store a piece of content with key `content-key`, a node performs a lookup to find the `k` closest nodes with radii that contain `content-id`. Then the node sends `Offer` messages to those nodes. For any node that responds to the `Offer` message with an `Accept` message, the local node attempts to transmit the content over the uTP connection with the `connection-id` from the `Accept` message.
```

Reply:
I don't think we would want to change `content-id` to `content-key` here. The naming is unfortunate but `content-id` is the "key" for the data in the DHT as described in the "content: keys and values" section.

Reply:
> I don't think we would want to change `content-id` to `content-key` here. The naming is unfortunate but `content-id` is the "key" for the data in the DHT as described in the "content: keys and values" section.

Right, I get it now that you mean DHT key, this naming is indeed unfortunate and confusing. I know it is somewhat explained above but could still be repeated perhaps like this then:

```suggestion
To store a piece of content with DHT key `content-id`, a node performs a lookup to find the `k` closest nodes with radii that contain `content-id`. Then the node sends an `Offer` message to those nodes. For any node that responds to the `Offer` message with an `Accept` message, the local node attempts to transmit the content over the uTP connection with the `connection-id` from the `Accept` message.
```